Scripts/Core/Contexts/EditingContextManager.cs

```diff
@@ -0,0 +1,282 @@
+﻿#if UNITY_EDITOR && UNITY_EDITORVR
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor.Experimental.EditorVR.Utilities;
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.InputNew;
+using System.IO;
+
+namespace UnityEditor.Experimental.EditorVR.Core
+{
+	class EditingContextManager : MonoBehaviour
```

Comment:
I'd prefer not to rename it because it sits within the EditorVR.Core namespace and I think is clear from that.